### ISSUE TITLE
logarithmic scale in line chart

### ISSUE THREAD

**Issue by siteswapper** [MusikAnimal/pageviews#35]
A logarithmic scale in the line chart diagram would be very beneficial to display the access statistics of the pages of, say, "Donald Trump", "Super Tuesday" and "Hillary Clinton" at the same time.

**Comment by MusikAnimal** [comment]
@siteswapper You mean show data on multiple articles at once? Just type them into the "Pages" input field! Or maybe I'm not understanding you

**Comment by MusikAnimal** [comment]
Such as http://tools.wmflabs.org/pageviews/#start=2016-02-20&end=2016-03-10&project=en.wikipedia.org&platform=all-access&agent=user&pages=Donald_Trump|Super_Tuesday|Hillary_Clinton

**Comment by siteswapper** [comment]
I got that I can have up to 10 articles simultaneously. But if one of them has very high access numbers, the others are pushed to the bottom and don't tell much of a story.
The same happens with only one graph, if part of the data point are very large. On this case, the smaller points seem so close to each other that they form just a straight line.
A solution is to draw the logarithms of the access numbers. This way all the graphs are clearly visible.
For more information, check out the Wikipedia article (semi-log plot)[https://en.m.wikipedia.org/wiki/Semi-log_plot].

**Comment by MusikAnimal** [comment]
Ah, I see. Unfortunately we're at the mercy of our Chart.js library, and I don't think it offers this functionality. I will keep this issue open, however, in case we decide to use another library. Thanks for the report.

**Comment by APerson241** [comment]
I agree with the user who opened this issue that this would be a great feature to have. I was just recently looking at the statistics for Merrick Garland, and his daily page view count jumped from 50, to 5,000, to one million in the span of weeks. A logarithmic scale would be incredibly useful for such occasions.

**Comment by MusikAnimal** [comment]
Looks like the newest (beta) version of Chart.js offers this! Will look into this soon.

**Comment by MusikAnimal** [comment]
See #42 for current blockers

**Comment by MusikAnimal** [comment]
Investigate some Chart.js 1.x patches, which do exist. It looks it's going to be a while before the blockers in #42 are resolved

**Comment by MusikAnimal** [comment]
This is being bundled in with the Charts 2.0 migration. See #42 for updates.

@nemobis You mentioned with #125 we could use the [Theil index](https://en.wikipedia.org/wiki/Theil_index) to automatically determine if we should use a logarithmic scale. I've read the article, and despite my self-proclaimed proficiency in mathematics, I can't make it sense of it. Do you have some simple equations or pseudocode to put it in layman's terms for me? :)

**Comment by MusikAnimal** [comment]
Done! :)
